game-core/src/main/java/games/strategy/ui/ImageScrollerSmallView.java

```diff
@@ -113,16 +116,25 @@ private void drawViewBox(final Graphics2D g) {
     final double width = model.getBoxWidth() * ratioX;
     final double height = model.getBoxHeight() * ratioY;
     final Rectangle2D.Double rect = new Rectangle2D.Double(x, y, width, height);
-    g.draw(rect);
+    drawFilledRect(rect, g);
     if (model.getScrollX()) {
       final double mapWidth = model.getMaxWidth() * ratioX;
       rect.x += mapWidth;
-      g.draw(rect);
+      drawFilledRect(rect, g);
       rect.x -= 2 * mapWidth;
-      g.draw(rect);
+      drawFilledRect(rect, g);
```

Comment:
General Note:
Here's the cause for a display bug for really low zoom settings.
If the minimap is smaller than the width of the panel behind it, the rectangles are actually being drawn outside of the map making it look weird.
Checking if the rectangle lies within the borders of the map would be good

Reply:
![Bug](https://user-images.githubusercontent.com/8350879/41200746-f86c1138-6caa-11e8-9dad-14fa2e011a37.png)

Reply:
@ron-murhammer  #3457